apps/cowswap-frontend/src/modules/hooksStore/dapps/WithdrawCoWAMMApp/hooks/usePoolUserBalance.tsx

```diff
@@ -0,0 +1,19 @@
+import { Token } from '@uniswap/sdk-core'
+
+import { PoolBalance } from '../types'
+
+export function usePoolUserBalance(poolId?: string): PoolBalance[] {
+  if (!poolId) return []
+  return [
+    {
+      token: new Token(1115511, '0x912ce59144191c1204e64559fe8253a0e49e6548', 18, 'ARB', 'Arbitrum'),
+      fiatAmount: '2765252315984615',
+      balance: '100000000000000000',
```

Comment:
🤔 maybe I don't have context here, but I assume we hardcoding this data here for testing purposes, right? Should we add a comment to come back to this in the future and remove it?